appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/jmx/MBeanMetadata.java

```diff
@@ -243,13 +240,13 @@ public MetricType getTypeRaw() {
     }
 
     public void setTags(List<XmlTag> tags) {
+        if (tags == null) {
```

Comment:
By removing this null check you allow the tags list to be set null. This later will cause NPE with no connection to the point where this null tags list came from. Rather keep this check to fail fast.

Reply:
"Rather keep this check to fail fast"?
I added the check -- exactly for the reason you mention.

Reply:
You are right